src/bindings/jna/README.md

```diff
@@ -50,33 +52,63 @@ Using the bindings via the command line works the same way as in Linux. The
 `/usr/local/share/java`.
 
-### Maven
+### Using a build system
 
-To use the jna bindings via maven, first you have to install the jna bindings
-to your local maven repository. When you have built Elektra with the jna
-bindings included, they should have been automatically installed to
-´/usr/share/java/´ along with a pom file for the library. To install it to your
-local maven repository from that location, execute the following command
-(Note: use `/usr/local/...` if it was installed there):
+To use the JNA bindings via a build system supporting Maven dependencies, you can depend on a released version published to Maven Central (available since version 0.9.5).
 
-```sh
-mvn org.apache.maven.plugins:maven-install-plugin:install-file \
-    -Dfile=/usr/share/java/libelektra5j.jar \
-    -DpomFile=/usr/share/java/libelektra5j.pom.xml
-```
+Alternatively you can install the JNA bindings to your local Maven repository (see instructions below).
+
+When you have built Elektra with the JNA binding included, it will also be automatically installed to
+`/usr/share/java/` along with a pom file for the library.
 
-Then you can simply add libelektra as dependency using:
+#### Maven
+
+Simply add libelektra as dependency using:
 
 ```xml
     <dependency>
       <groupId>org.libelektra</groupId>
-      <artifactId>libelektra5j</artifactId>
-      <version>0.9.4</version>
+      <artifactId>libelektra</artifactId>
+      <version>0.9.5</version>
     </dependency>
 ```
 
-[Here](../../examples/external/java/read-keys-example/pom.xml) is a full example using maven,
-which should work out of the box if above `mvn` command was executed.
+[Here](../../../examples/external/java/read-keys-example/pom.xml) is a full example using Maven, which should work out of the box by executing `mvn compile exec:java` in the example directory `../../../examples/external/java/read-keys-example/`.
+
+#### Gradle
+
+Simply add libelektra as dependency to you Gradle `java` or `java-library` project using:
+
+```groovy
+repositories {
+    mavenLocal()
+    mavenCentral()
+}
+
+dependencies {
+    implementation "org.libelektra:libelektra:0.9.5"
+}
+```
+
+[Here](../../../examples/external/java/read-keys-example/build.gradle) is a full example using Gradle, which should work out of the box by executing `gradle run` in the example directory `../../../examples/external/java/read-keys-example/`.
+
+### Installing the JNA binding to the local Maven repository
+
+Release versions of the Elektra JNA bindings are published to Maven Central with group id `org.libelektra`, artefact id `libelektra` and the same version as Elektra.
+
+If you want to depend on a modified binding or just want to install it to your local maven repository, change your current working directory to the JNA binding folder `/src/bindings/jna`. Either use the bundled Gradle wrapper (`./gradlew` for Unix style OS or `./gradlew.bat` for windows) or ensure a recent Gradle version is available on your system and execute:
```

Comment:
Is it recommended that we keep both the wrapper and also installed versions of gradle, or should we get rid of one of the two now?

Reply:
Developers might have different Gradle versions installed. The wrapper is the recommended way to get consistent Gradle versions AFAIK.

Within CI and Docker it might be problematic (i.e. a waste of time) to let Gradle Wrapper download the Gradle binaries every time. If this can be avoided (e.g. via caching or multi-stage Docker builds), I think it would be preferable to use Gradle Wrapper everywhere. If this is not possible, then we should probably list all the places where Gradle Version is defined in a central location.

Reply:
PS. for in the README I would only recommend using Gradle Wrapper, since that gives a consistent environment